.pre-commit-config.yaml

```diff
@@ -35,9 +35,10 @@ repos:
         additional_dependencies:
           - "backoff>=2.2.1,<3"
           - "click>=8.1.7,<9"
+          - "jinja2>=3.1.2,<4"
           - "pandas-stubs>=2.1.1,<3"
           - "pydantic-settings>=2.1.0,<3"
-          - "pydantic>=2.5.2,<3"
+          - "pydantic[email]>=2.5.2,<3"
```

Comment:
pre-commit gives me this error `site-packages\pydantic\networks.py:18: error: Cannot find implementation or library stub for module named "email_validator"  [import-not-found]` otherwise

Reply:
alrighty, will be gone with MX-1515 anyway.